.github/workflows/ci-tests.yml

```diff
@@ -39,10 +39,6 @@ jobs:
       # Make sure we use the latest version of MatrixKit
       - name: Reset MatrixKit pod
         run: rm -rf Pods/MatrixKit
-        
-      # Make sure we use the latest version of MatrixSDK
-      - name: Reset MatrixSDK pod
-        run: rm -rf Pods/MatrixSDK
```

Comment:
idem - same comment

Reply:
Changes here are just to clean the cache for CI.
The reset MatrixKit will be replaced by reset MatrixSDK with the next rebase.